executor/join_test.go

```diff
@@ -89,9 +89,12 @@ func (s *testSuite) TestNestedLoopJoin(c *C) {
 func (s *testSuite) TestJoinPanic(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
+	tk.MustExec("set sql_mode = 'ONLY_FULL_GROUP_BY'")
```

Comment:
Does it still panic if `ONLY_FULL_GROUP_BY` is not set?

Reply:
it would not panic.